Helper.java

```diff
@@ -99,6 +108,35 @@ static void enterData() throws InvalidAnimalName {
                     }
                 }
                 case 3 -> {
+                    for (Map.Entry<String, List<Animal>> mp : animalSpecies.entrySet()) {
+                        if (mp.getKey().equals("dog")) {
+                            System.out.println(mp.getValue());
+                        }
+                    }
+                }
+                case 4 -> {
+                    for (Map.Entry<String, List<Animal>> mp : animalSpecies.entrySet()) {
+                        if (mp.getKey().equals("cat")) {
+                            System.out.println(mp.getValue());
+                        }
+                    }
+                }
+                case 5 -> {
```

Comment:
Let`s merge 3-5 to 1 - too much duplicates.
Imagine if we have 1000 Types of inctances.
It would be really inconvenient to maintain it.
In our case we can replace it with 1 general method